app/component/ItineraryPageUtils.js

```diff
@@ -358,6 +360,8 @@ export function checkDayNight(iconId, timem, lat, lon) {
   return iconId;
 }
 
+const streetLegModes = ['WALK', 'BICYCLE', 'CAR', 'SCOOTER'];
```

Comment:
Spelling of these static helper variables could be STREET_LEG_MODES but not that important.

Reply:
done